hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/records/RecordsHistorian.java

```diff
@@ -98,6 +98,21 @@ public interface RecordsHistorian {
      */
     List<RecordStreamObject> getPrecedingChildRecords();
 
+    /**
+     * For each following child transaction which:
+     * <ul>
+     *   <li>Is <i>not</i> a {@code ContractCall} or {@code ContractCreate} operations, consumes
+     *   capacity in the consensus throttle; and,</li>
+     *   <li>Has status {@code SUCCESS}</li>
+     * </ul>
+     * consumes the capacity required for that child transaction in the consensus throttle buckets.
+     *
+     * <p>Returns true if all the child transactions were successfully throttled, false otherwise.
+     *
+     * @return whether the consensus throttle has capacity to allow all the successful 0x167 children
```

Comment:
What is `0x167`?

Reply:
It where the precompile contracts are located.